applications/POMP.md

```diff
@@ -0,0 +1,90 @@
+
+The front-end website:
+
+![](https://raw.githubusercontent.com/pomp-money/pomp-documents/main/designs/mint.png)
+![](https://raw.githubusercontent.com/pomp-money/pomp-documents/main/designs/mint-options.png)
+![](https://raw.githubusercontent.com/pomp-money/pomp-documents/main/designs/minted.png)
+
+
+### Ecosystem Fit
+
+To prove ownership of DOT or other Polkadot Parachain cryptocurrencies, we need to prove that we know the private key. However, we would not want to reveal the private key, as revealing even part of the private key or an image of the private key could put our ownership at risk. By having the private key — or even part of it and then using sophisticated software to attempt to guess the rest of it — hackers can steal your cryptocurrency. 
+
+The most reliable way to prove ownership of cryptocurrencies is to sign a specified message with your private key. By doing so, you can prove ownership of your crypto assets without the need to reveal the key or having to engage in a transaction, even a small amount transfer.
+
+The POMP platform enables all crypto token owners of Polkadot Parachain to generate their encrypted proof key through zero knowledge and prove crypto assets without the risk that may be leaking the private key.
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Megan
+- zqhxuyuan
+- Chan
+
+### Contact
+
+- **Contact Name:** Megan
+- **Contact Email:** megan.zhang76@gmail.com
+
+### Team's experience
+
+We are a decentralized team focusing on the blockchain industry since 2020. We are deep participants in several technical communities and hackathon winners. 
+
+### Team Code Repos
+
+- https://github.com/zhangjing28
+- https://github.com/jincan39
+- https://github.com/zqhxuyuan
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 2 months
+- **Full-Time Equivalent (FTE):**  3 FTE
+- **Total Costs:** 30,000 USD
+
+### Milestone 1 — The POMP platform
+
+- **Estimated duration:** 2 month
+- **FTE:**  3
+- **Costs:** 30,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user sends test transactions, which will show how the new platform works. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 0d. | Article | We will publish an **article** that explains what this grant achieved.|
+| 1. | Javascript Library | We will deliver a JS library that will implement the functionality described under the "Workflow" part, including recovering the address from the signature, verifying the asset amount, hashing the ZK proof key, and integrating it with the ink!  smart contract functions.|
```

Comment:
Will other devs be able to use this library to integrate the tool into their dApp? Or does the user essentially have to "opt in" by using Manta wallet to generate the proof?

Reply:
Sure, any other devs are able to integrate our JS library to verify the proof key that is saved on ink! smart contract. And we have to use the Manta wallet to generate the zero-knowledge data when the user would like to create Proof of Possession. Which means generate once, verify anywhere.